gulpfile.js

```diff
@@ -32,6 +33,46 @@ const allPackages = [];
  */
 const namespace = process.argv.includes('--drop-in') ? '' : 'pwa-';
 
+async function runBuildScript(name) {
```

Comment:
Could either `runBuildScript` or `readJson` be simplified with `util.promisify`? They're both marked as `async` but use no awaits.

Reply:
readJson can, but util.promisify doesn't support the multiple callback items that execFile gives back to us